composer.json

```diff
@@ -12,7 +12,7 @@
     "require": {
         "php": "^5.6|^7.0",
         "symfony/framework-bundle": "^2.8|^3.0",
-        "symfony-cmf/routing-auto": "2.0.0-RC2",
+        "symfony-cmf/routing-auto": "dev-jms-metadata-version as 2.0.0-RC3",
```

Comment:
@dbu RC4? That is my last release, i think we should put your changes into a RC5?